sympy/physics/units/dimensions.py

```diff
@@ -572,3 +572,74 @@ def is_consistent(self):
         # dimensions
         # in vector language: the set of vectors do not form a basis
         return self.inv_can_transf_matrix.is_square
+    def getDimensionlessNumbers(self, list_of_derived_Quantities 
+        # input list of derived quantities in the
+        # form of a python dictionary of tuples (key, item) where 'key' is
+        # the name of the physical quantity and 'item' is an instance of
+        # Dimension class which possesses the dimensions of the quantity.
+        ):
+        """ This function is designed to take in a set of physical variables (all 
+    	essentially dimensionful) and returns one set of all possible dimensionless
+	    quantities, as guided by the Buckingham's pi theorem (a generalization of 
+	    Rayleigh's method of dimensional analysis .....)
+	    
+        Please refer and run to the unit testing file '/units/test/test_dimensionLess.py' for
+            the format of the input .... As a heads up, we first need to define
+            a global set of base dimensions and with those we can define the 
+            derived quantities .....
+        """
+        ExponentMatrix = []
```

Comment:
In SymPy, we are trying to use *CamelCase* style for class names, we prefer *snake_case* for variable names.

It's not mandatory, but having a unified style makes the code more readable.

Reply:
@Upabjojr correct, makes sense! I'll correct the variables in the future edits.